### PR TITLE
Allow external compile type to use PATH executable as input

### PR DESCRIPTION
**Pull request by ofen**
Fix for https://github.com/kapicorp/kapitan/issues/695

### CODE REVIEW

**Comment by ofen** [comment]
@ramaro could you please take a look

File: docs/compile.md [hunk 1]
```diff
@@ -552,15 +552,15 @@ parameters:
     compile:
       - input_type: external
         input_paths:
-          - /usr/local/bin/ytt # path to ytt on system
-        output_path: .
+          - ytt # path to ytt on system
+        output_path: ingresses/ingresses.yaml
```
Comment (ramaro) [inline docs/compile.md:556]: Why are we changing this?

**Comment by ramaro** [comment]
Hey @ofen sorry for the delay and thanks for this. I have a question - wouldn't this break compatibility with the current behavior? If so, I think a better option would be to have a new  parameter, e.g. `in_path: true` to enable this.

**Comment by simu** [comment]
I think this should be addresssed in #864

**Comment by ramaro** [comment]
> I think this should be (at least partially) addressed in #864

Yes it is. Thanks. closing.

**Pull request closed without merging**